CandyMarket/CandyMarket/Controllers/CandyController.cs

```diff
@@ -54,7 +54,19 @@ public IActionResult AddNewCandy(int userId, Candy candyToAdd)
                 return Ok(candyToAdd);
             }
             return NotFound("That user does not exist.");
+        }
+
 
+        // api/Candy/eatCandy/2/user/3
+        [HttpDelete("eatCandy/{candyId}/user/{userId}")]
+        public IActionResult ConsumeChosenCandy(int candyId, int userId)
+        {
+            var candyConsumed = _repository.ConsumeSpecificCandy(candyId, userId);
+            if (candyConsumed == null)
+            {
+                return NotFound("This candy could not be found for the specified user");
```

Comment:
Do we want to wrap this to check that the user is valid to return a different error?  Otherwise if incorrect user input then it returns the candy couldn't be found for specified user.

Reply:
done!